docs/java-rest/high-level/getting-started.asciidoc

```diff
@@ -6,7 +6,7 @@ getting the artifact to using it in an application.
 
 [[java-rest-high-compatibility]]
 === Compatibility
-The Java High Level REST Client requires Java 1.8 and depends on the Elasticsearch
+The Java High Level REST Client requires at least Java 1.8 and depends on the Elasticsearch
```

Comment:
As a java developer, when ever I see a Java version I almost always assume "at least" given Java's good history of compatibility with newer versions. I am not sure if this addition provides much value...but that could just be my personal opinion.  

@swallez - thoughts here ?

Reply:
@jakelandis as a Java old timer I share your sentiment, but also understand that Java's great track record regarding compatibility may not be universally known. So let's make it more explicit with this change.

And hopefully this will also encourage people to use a more modern version than the antiquated Java 1.8 😉

Reply:
Happy to see this fix, which closes #63525 😃